src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -1140,6 +1141,7 @@ protected static void VerifyFlowGraphForTest<TSyntaxNode>(CSharpCompilation comp
                 map.Add(graph[i], i);
             }
 
+            var visitor = TestOperationVisitor.GetInstance();
```

Comment:
Not a problem with this PR, but consider renaming `GetInstance()` to `GetSingleton()` or `Singleton`. 
From the dozens of methods I checked, `GetInstance` methods in the codebase typically allocate something that should be freed.

Reply:
> Not a problem with this PR, but consider renaming GetInstance() to GetSingleton() or Singleton. 
From the dozens of methods I checked, GetInstance methods in the codebase typically allocate something that should be freed.

I'll address this in the next PR, if that is Ok.

---
In reply to: [163070185](https://github.com/dotnet/roslyn/pull/24358#discussion_r163070185) [](ancestors = 163070185)